src/toast/instrument_sim.py

```diff
@@ -230,16 +234,14 @@ def hex_layout(npos, angwidth, prefix, suffix, pol, center=None, pos_offset=0):
     rtthree = np.sqrt(3.0)
     rtthreebytwo = 0.5 * rtthree
 
-    # compute the diameter (vertex to vertex width)
-    angdiameter = angwidth.to_value(u.radian) / np.cos(thirty)
+    # The diameter (vertex to vertex width)
+    angdiameter = angwidth.to_value(u.radian)
```

Comment:
This looks like a big change. What happened here?

Reply:
The issue is that in some downstream code this layout function was being called with the vertex-vertex width, not the width of the "flats".  That is actually the quantity that is easiest to define since it is a whole number of pixel spacings.  So I am bringing all 3 codes into line with that definition.  So yes, this is a change to the definition and cannot be merged until `sotodlib` and `s4sim` have corresponding PRs with any needed changes for consistency.